### ISSUE TITLE
Missing info in body for Docker APIv2 push endpoint

### ISSUE THREAD

**Issue by matejvasek** [containers/podman#9277]
Podman returns 200 and empty body upon successful push.
Docker however returns 200 and non empty body.
The body looks like:
```
{"status":"The push refers to repository [localhost:5000/myrepo/myimg]"}
{"status":"Preparing","progressDetail":{},"id":"1119ff37d4a9"}
{"status":"Pushing","progressDetail":{"current":68608,"total":5613130},"progress":"[\u003e                                                  ]  68.61kB/5.613MB","id":"1119ff37d4a9"}
{"status":"Pushing","progressDetail":{"current":1558528,"total":5613130},"progress":"[=============\u003e                                     ]  1.559MB/5.613MB","id":"1119ff37d4a9"}
{"status":"Pushing","progressDetail":{"current":3824128,"total":5613130},"progress":"[==================================\u003e                ]  3.824MB/5.613MB","id":"1119ff37d4a9"}
{"status":"Pushing","progressDetail":{"current":5879808,"total":5613130},"progress":"[==================================================\u003e]   5.88MB","id":"1119ff37d4a9"}
{"status":"Pushed","progressDetail":{},"id":"1119ff37d4a9"}
{"status":"mytag: digest: sha256:3747d4eb5e7f0825d54c8e80452f1e245e24bd715972c919d189a62da97af2ae size: 528"}
{"progressDetail":{},"aux":{"Tag":"mytag","Digest":"sha256:3747d4eb5e7f0825d54c8e80452f1e245e24bd715972c919d189a62da97af2ae","Size":528}}
```

**Comment by matejvasek** [comment]
I would like to at least receive digest.

**Comment by rhatdan** [comment]
Not sure if this is a bug or a feature?

**Comment by matejvasek** [comment]
@rhatdan neither am I.

I am not sure whether this is mandatory of just for user convenience (progress displaying). However docker does output it.

Unfortunately [one of our tools](https://github.com/boson-project/func/) relies on fact that output contains `digest: sha256:\w{64}` , IIRC at the time we didn't found better way to receive digest upon `push`.

I am already looking into fix.

For `/libpod/` endpoints no change is needed, however we might consider writing digest in response (instead of empty string) if anybody found it useful.

**Comment by rhatdan** [comment]
Interested in opening a PR for this?

**Comment by mlegenovic** [comment]
#8713

**Comment by matejvasek** [comment]
I will, but there is a dependency on another PR of mine. I'll create PR for this once the other PR is merged.

**Comment by mlegenovic** [comment]
And the same problem just for pull endpoint #7857

**Comment by rhatdan** [comment]
Looks like this has been fixed in the main branch.
